docs/product-administration-and-configuration-guide/src/main/asciidoc/kie-server-accessing-runtime-data-proc.adoc

```diff
@@ -0,0 +1,15 @@
+[id='kie-server-accessing-runtime-data-proc']
+= Accessing Runtime Data from {CENTRAL}
+
+The `Process Instances`, `Tasks`, and `Process & Task Dashboard` pages of Business Central allow you to view the runtime data of the Intelligent Process Server. These runtime pages use the credentials of the currently logged in user to load data from the Intelligent Process Server. Therefore, to be able to view the runtime data on Business Central, ensure that the following conditions are met:
```

Comment:
Aren't we calling Business Central as "Decision Central"?
I may be wrong, but isn't "Intelligent Process Server" a server in BA context and not DM?

Reply:
I thinks now it's just "KIE Server" - please see https://docs.google.com/document/d/1vypMG1ple4OXG9krcZ5-VbsQm_AGCOap_7pe6pMI4qs

Reply:
This topic isn't included in the assembly. Sorry, it's just in the PR due to the modularization required on this chapter to use parts of it.